pyuvm/utility_classes.py

```diff
@@ -122,13 +122,13 @@ def find_override(self, requested_type, inst_path=None, overridden_list=None):
 
         # xyz -> foo -> bar
         #
-        # So if find_override is fo:
-        #     fo(xyz) -> fo(foo) -> fo(bar) <-- no override returns bar.
+        # So if find_override is f:
+        #     f(xyz) -> f(foo) -> f(bar) <-- no override returns bar.
         # Recursive loops result ina n error in which case the
         # type returned is the one that formed the loop:
         # xyz -> foo -> bar -> xyz
         #
-        # fo(xyz) -> fo(foo) -> fo(bar) -> fo(xyz) -- xyz is in
```

Comment:
This is codespell being pretty pedantic, but it seemed better to change it rather than add "fo" to the ignore list and miss checking misspellings of "of".